src/site/pages/digital/src/containers/ExprToCircuitPopup/generate.ts

```diff
@@ -105,7 +105,7 @@ export function Generate(info: DigitalCircuitInfo, expression: string,
                          userOptions: Partial<ExprToCirGeneratorOptions>) {
     const options = {...defaultOptions, ...userOptions};
     options.isIC = (options.output !== "Oscilloscope") ? options.isIC : false;
-    const ops = options.format === "custom" ? options.ops : Formats.find(form => form.icon === options.format);
+    const ops = options.format === "custom" ? options.ops : Formats.find(form => form.icon === options.format) ?? Formats[0];
```

Comment:
Put some parenthesis somewhere here, `a ? b : c ?? d` looks kinda strange

Reply:
Originally thought of this as `(a ? b : c) ?? d` but in this case I think `a ? b : (c ?? d)` makes more sense so I set it to that